app/dossierPrograms/dossierPrograms.services.js

```diff
@@ -2,25 +2,20 @@
     List of contributors: https://github.com/MSFOCBA
     Please refer to the LICENSE.md and LICENSES-DEP.md for complete licenses.
 ------------------------------------------------------------------------------------*/
-var qryPrograms = dhisUrl + "programs.json?fields=id,displayName,displayDescription,programStages[id]&paging=false";
+var qryPrograms = dhisUrl + "programs?fields=id,displayName,displayDescription,programStages[id]&paging=false";
 
 var qryProgramStageSections =
     dhisUrl +
-    "programStages/:programStageId.json?fields=id,displayName,repeatable,sortOrder,programStageSections[id,displayName,dataElements[displayName,displayFormName,displayDescription,valueType,optionSetValue,optionSet[options[displayName]]]],programStageDataElements[dataElement[displayName,displayFormName,displayDescription,valueType,optionSetValue,optionSet[options[displayName]]]]&paging=false";
-
-var qryProgramIndicators =
-    dhisUrl + "programs/:programId.json?fields=programIndicators[displayName,displayDescription,expression,filter]";
-
-var qryProgramIndicatorExpressions = dhisUrl + "programIndicators/expression/description";
-var qryProgramIndicatorFilters = dhisUrl + "programIndicators/filter/description";
+    "programStages/:programStageId?fields=id,displayName,repeatable,sortOrder,\
```

Comment:
[comment] I see that this long string-concatenation styles come from the original code, but now that we have to change it, some proposals:

1) [simple] Use  [field1, field2, ...].join(",") (directly or with a thin wrapper).
2) More complex, wrapper d2-api style, that converts to string something like `params({id: true, name: true, options: {id: true, code: true}})`.  (complex, a probably a bit overkill here)

Reply:
Switched to the first proposal.